pywbemtools/pywbemlistener/_cmd_listener.py

```diff
@@ -0,0 +1,1176 @@
+    def indi_file(self):
+        """string: Append each received indication to a file in a format"""
+        return self._indi_file
+
+    @property
+    def indi_format(self):
+        """string: Set format of indication"""
+        return self._indi_format
+
+    @property
+    def logfile(self):
+        """string: Path name of log file"""
+        return self._logfile
+
+    @property
+    def pid(self):
+        """int: Process ID of the listener"""
+        return self._pid
+
+    @property
+    def created(self):
+        """datetime: Point in time when the listener process was created"""
+        return self._created
+
+
+@cli.command('run', cls=PywbemtoolsCommand, options_metavar=CMD_OPTS_TXT)
+@click.argument('name', type=str, metavar='NAME', required=False)
+@add_options(LISTEN_OPTIONS)
+@add_options(help_option)
+@click.pass_obj
+def listener_run(context, name, **options):
+    """
+    Run as a named WBEM indication listener.
+
+    Run this command as a named WBEM indication listener until it gets
+    terminated, e.g. by a keyboard interrupt, break signal (e.g. kill), or the
+    `pywbemlistener stop` command.
+
+    A listener with that name must not be running, otherwise the command fails.
+
+    Examples:
+
+      pywbemlistener run lis1
+    """
+    if show_help_options(options):
+        return
+    validate_required_arg(name, 'NAME')
+    context.execute_cmd(lambda: cmd_listener_run(context, name, options))
+
+
+@cli.command('start', cls=PywbemtoolsCommand, options_metavar=CMD_OPTS_TXT)
+@click.argument('name', type=str, metavar='NAME', required=False)
+@add_options(LISTEN_OPTIONS)
+@add_options(help_option)
+@click.pass_obj
+def listener_start(context, name, **options):
+    """
+    Start a named WBEM indication listener in the background.
+
+    A listener with that name must not be running, otherwise the command fails.
+
+    A listener is identified by its hostname or IP address and a port number.
+    It can be started with any free port.
+
+    Examples:
+
+      pywbemlistener start lis1
+    """
+    if show_help_options(options):
+        return
+    validate_required_arg(name, 'NAME')
+    context.execute_cmd(lambda: cmd_listener_start(context, name, options))
+
+
+@cli.command('stop', cls=PywbemtoolsCommand, options_metavar=CMD_OPTS_TXT)
+@click.argument('name', type=str, metavar='NAME', required=True)
+@add_options(help_option)
+@click.pass_obj
+def listener_stop(context, name):
+    """
+    Stop a named WBEM indication listener.
+
+    The listener will shut down gracefully.
+
+    A listener with that name must be running, otherwise the command fails.
+
+    Examples:
+
+      pywbemlistener stop lis1
+    """
+    context.execute_cmd(lambda: cmd_listener_stop(context, name))
+
+
+@cli.command('show', cls=PywbemtoolsCommand, options_metavar=CMD_OPTS_TXT)
+@click.argument('name', type=str, metavar='NAME', required=True)
+@add_options(help_option)
+@click.pass_obj
+def listener_show(context, name):
+    """
```

Comment:
It would be really interesting to see the number of indications received and time of last with the show command.   At this point I would be willing to put that off to future.  It would be nice to have such statistics defined for each listener and available to the CALL function

Reply:
that should be a feature left for a future PR.